tracer/src/Datadog.Trace/AppSec/BlockException.cs

```diff
@@ -27,5 +27,8 @@ internal BlockException(System.Runtime.Serialization.SerializationInfo info, Sys
             : base(info, context)
         {
         }
+
+        // can give a significant performance boost, this exception is currently caught and logged by the host web server
+        public override string ToString() => "BlockException";
```

Comment:
NIT - Could have been nice to have it in a separate PR to see the impact on the throughput tests once merged.

Reply:
fixed in 270880fff